<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditTransactionDialog</class>
 <widget class="QDialog" name="EditTransactionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit transaction</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_8">
   <item>
    <layout class="QHBoxLayout" name="hlContent">
     <item>
      <widget class="QWidget" name="wCalendarContainer" native="true">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,1">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="labelTransactionType">
          <property name="text">
           <string>Transaction type:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="wTypesContainer" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="bTypeOutgoing">
             <property name="text">
              <string>Outgoing</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">typesButtonGroup</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="bTypeIncoming">
             <property name="text">
              <string>Incoming</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">typesButtonGroup</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="bTypeInternal">
             <property name="text">
              <string>Internal</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">typesButtonGroup</string>
             </attribute>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelPerformedAt">
          <property name="text">
           <string>Performed at:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="hlCalendarEditButton">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QDateTimeEdit" name="eDatetime">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bSetCurrentDate">
            <property name="text">
             <string>Set current</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCalendarWidget" name="wCalendar"/>
        </item>
        <item>
         <spacer name="vsCalendar">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="Line" name="lineContent">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="swContent">
       <widget class="QWidget" name="pageOutgoing">
        <layout class="QVBoxLayout" name="verticalLayout_7" stretch="0,0,0,1">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="hlAccountAmount">
           <item>
            <layout class="QVBoxLayout" name="vlAccount">
             <item>
              <widget class="QLabel" name="labelAccountOutgoing">
               <property name="text">
                <string>Account:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="cbAccountOutgoing">
               <property name="sizePolicy">
                <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="vlAmount">
             <item>
              <widget class="QLabel" name="labelAmountOutgoing">
               <property name="text">
                <string>Amount:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="sbAmountOutgoing">
               <property name="minimumSize">
                <size>
                 <width>80</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximum">
                <double>10000000.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="vlCategory">
           <item>
            <widget class="QLabel" name="labelCategoryOutgoing">
             <property name="text">
              <string>Category:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cbCategoryOutgoing"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="vlNotes">
           <item>
            <widget class="QLabel" name="labelNotesOutgoing">
             <property name="text">
              <string>Notes:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPlainTextEdit" name="eNotesOutgoing">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>40</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="vlSubitems">
           <item>
            <layout class="QHBoxLayout" name="hlSubitemsButtons">
             <property name="spacing">
              <number>1</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QCheckBox" name="cSubitems">
               <property name="text">
                <string>Has sub items</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="hsSubitemsButtons">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="bSubitemAdd">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="maximumSize">
                <size>
                 <width>22</width>
                 <height>22</height>
                </size>
               </property>
               <property name="text">
                <string>+</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="bSubitemDelete">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="maximumSize">
                <size>
                 <width>22</width>
                 <height>22</height>
                </size>
               </property>
               <property name="text">
                <string>-</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTableWidget" name="twSubitems">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::SingleSelection</enum>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
             <property name="verticalScrollMode">
              <enum>QAbstractItemView::ScrollPerPixel</enum>
             </property>
             <property name="horizontalScrollMode">
              <enum>QAbstractItemView::ScrollPerPixel</enum>
             </property>
             <attribute name="horizontalHeaderCascadingSectionResizes">
              <bool>true</bool>
             </attribute>
             <attribute name="horizontalHeaderHighlightSections">
              <bool>false</bool>
             </attribute>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <attribute name="verticalHeaderDefaultSectionSize">
              <number>20</number>
             </attribute>
             <column>
              <property name="text">
               <string>Name</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Category</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Amount</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="pageIncoming">
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="vlDestinationAccount">
           <item>
            <widget class="QLabel" name="labelDestinationAccount">
             <property name="text">
              <string>Destination account:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cbDestinationAccountIncoming"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="vlAmountIncloming">
           <item>
            <widget class="QLabel" name="labelAmountIncoming">
             <property name="text">
              <string>Amount:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="sbAmountIncoming">
             <property name="minimum">
              <double>-10000000.000000000000000</double>
             </property>
             <property name="maximum">
              <double>1000000.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="vlCategoryIncoming">
           <item>
            <widget class="QLabel" name="labelCategoryIncoming">
             <property name="text">
              <string>Category:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cbCategoryIncoming"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="vlNotesIncoming">
           <item>
            <widget class="QLabel" name="labelNotesIncoming">
             <property name="text">
              <string>Notes:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPlainTextEdit" name="eNotesIncoming"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="pageInternal">
        <layout class="QVBoxLayout" name="verticalLayout_13">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="vlSourceAccountInternal">
           <item>
            <widget class="QLabel" name="labelSourceAccountInternal">
             <property name="text">
              <string>Source account:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cbSourceAccountInternal"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="vlDestinationAccountInternal">
           <item>
            <widget class="QLabel" name="labelDestinationAccountInternal">
             <property name="text">
              <string>Destination account:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cbDestinationAccountInternal"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="vlAmountInternal">
           <item>
            <widget class="QLabel" name="labelAmountInternal">
             <property name="text">
              <string>Amount:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="sbAmountInternal">
             <property name="minimum">
              <double>-10000000.000000000000000</double>
             </property>
             <property name="maximum">
              <double>10000000.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="vlNotesInternal">
           <item>
            <widget class="QLabel" name="labelNotesInternal">
             <property name="text">
              <string>Notes:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPlainTextEdit" name="eNotesInternal"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="lineMain">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlFooter">
     <item>
      <spacer name="hsFooter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="bCancel">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bOk">
       <property name="text">
        <string>&amp;OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>wCalendar</sender>
   <signal>clicked(QDate)</signal>
   <receiver>eDatetime</receiver>
   <slot>setDate(QDate)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>104</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>59</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>eDatetime</sender>
   <signal>dateChanged(QDate)</signal>
   <receiver>wCalendar</receiver>
   <slot>setSelectedDate(QDate)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bSetCurrentDate</sender>
   <signal>clicked()</signal>
   <receiver>wCalendar</receiver>
   <slot>showToday()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>156</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bCancel</sender>
   <signal>clicked()</signal>
   <receiver>EditTransactionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>326</x>
     <y>326</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cSubitems</sender>
   <signal>toggled(bool)</signal>
   <receiver>twSubitems</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>330</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cSubitems</sender>
   <signal>toggled(bool)</signal>
   <receiver>sbAmountOutgoing</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>396</x>
     <y>42</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cSubitems</sender>
   <signal>toggled(bool)</signal>
   <receiver>bSubitemAdd</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel">
     <x>443</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cSubitems</sender>
   <signal>toggled(bool)</signal>
   <receiver>bSubitemDelete</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel">
     <x>466</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="typesButtonGroup"/>
 </buttongroups>
</ui>
